### PR TITLE
Simplify the tracker icon asset loading code 

### PR DESCRIPTION
**Pull request by miasma13**
Task/Issue URL: https://app.asana.com/0/414709148257752/1204033732341887/f

**Description**:
Background
The code for loading matching logo or first letter icon for blocked trackers animation was heavily based on original macOS implementation. It was build using hugely repetitive switch statement. Additionally the code contained typo and was not properly loading a dark mode variant for one of the logos. Since iOS implementation uses automatic support for loading dark mode assets we can simplify the code.

**Steps to test this PR**:
- Test if logo icons load for known trackers by their entity name.
- Ensure that icons for trackers containing space in their name ("the trade desk" and "verizon media") are loaded properly
- Check if trackers without matching logo load generic first letter icon instead

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

File: DuckDuckGo/TrackerImageCache.swift [hunk 1]
```diff
@@ -57,83 +57,12 @@ final class TrackerImageCache {
...
-        case "pubmatic": return UIImage(named: "pubmatic", in: nil, compatibleWith: trait)!.cgImage!
-        case "qwantcast": return UIImage(named: "qwantcast", in: nil, compatibleWith: trait)!.cgImage!
-        case "rubicon": return UIImage(named: "rubicon", in: nil, compatibleWith: trait)!.cgImage!
-        case "salesforce": return UIImage(named: "salesforce", in: nil, compatibleWith: trait)!.cgImage!
-        case "smartadserver": return UIImage(named: "smartadserver", in: nil, compatibleWith: trait)!.cgImage!
-        case "spotx": return UIImage(named: "spotx", in: nil, compatibleWith: trait)!.cgImage!
-        case "stackpath": return UIImage(named: "stackpath", in: nil, compatibleWith: trait)!.cgImage!
-        case "taboola": return UIImage(named: "taboola", in: nil, compatibleWith: trait)!.cgImage!
-        case "tapad": return UIImage(named: "tapad", in: nil, compatibleWith: trait)!.cgImage!
-        case "the trade desk": return UIImage(named: "thetradedesk", in: nil, compatibleWith: trait)!.cgImage!
```
Comment (jaceklyp) [inline DuckDuckGo/TrackerImageCache.swift:97]: what about it?

Comment (miasma13) [reply DuckDuckGo/TrackerImageCache.swift:97]: The asset was renamed to match the entity name so it is properly resolved.

**Comment by Bunn** [comment]
@jaceklyp @miasma13 anything left to do in this PR?

**Review by jaceklyp — APPROVED** [review]
LGTM!

**Pull request merged**